src/Homepage.js

```diff
@@ -42,7 +68,11 @@ const Homepage = () => {
     }
 
     useEffect(() => {
-        console.log('Entry was made')
+        setButton("On");
+        setSoundText("speech Off")
+        if (typeof window !== 'object' || !window.speechSynthesis) return;
```

Comment:
What is this for?

Reply:
So on load it will display The "On" and "speech Off"